### ISSUE TITLE
Bright vs Dim ANSI color codes

### ISSUE THREAD

**Issue by Aceeri** [slog-rs/term#2]
`slog-term` displays everything as a dimmed color code `[ 2 m` while rustc logs warnings and errors with a bright color code `[ 1 m`

Differences:
![Differences](http://image.prntscr.com/image/e5ccb37f4bd142408695409b96a1e13a.png)

![Example](http://image.prntscr.com/image/42ba363abab14d34ad408fed3e0163cf.png)

![Rustc Warning Example](http://image.prntscr.com/image/73b89fcd4a9b40c8be167a1ca6f4d611.png)

**Comment by dpc** [comment]
Can you submit a PR? :)

**Comment by dpc** [comment]
slog-term uses:

```
try!(write!(io, "\x1b[3{}m", level_color));
```

which according to http://wiki.bash-hackers.org/scripting/terminalcodes just changes the foreground color.

**Comment by dpc** [comment]
I briefly tried changing it to:

```
try!(write!(io, "\x1b[3{}m\x1b[1m", level_color));
```

and that gives not the color I want on Linux (`gnome-terminal`). I guess using different string literal for Windows and Unix could do the job.

**Comment by dpc** [comment]
After more discussion on gitter, we agreed that using `term` crate is the way to go, and it does not introduce any additional allocations, as it implements `io::Write` on `Terminal` https://stebalien.github.io/doc/term/term/trait.Terminal.html

**Comment by dpc** [comment]
I think this is fixed as there are people using it (see #8)
